package.json

```diff
@@ -22,14 +22,31 @@
   "devDependencies": {
     "autoprefixer": "^6.3.2",
     "cssnano": "^3.5.1",
+    "eslint": "^4.11.0",
     "gulp": "^3.9.0",
     "gulp-concat": "^2.6.0",
     "gulp-header": "^1.7.1",
     "gulp-postcss": "^6.1.0",
     "gulp-rename": "^1.2.2",
     "gulp-util": "^3.0.7",
+    "prettier": "^1.8.2",
     "run-sequence": "^1.1.5"
   },
+  "lint-staged": {
+    "*.{js,json,md,css}": ["prettier --write", "git add"]
+  },
+  "prettier": {
+    "bracketSpacing": false,
+    "printWidth": 120,
+    "proseWrap": false,
+    "singleQuote": true,
+    "tabWidth": 2,
+    "trailingComma": "all"
```

Comment:
Let me know if the rules are OK, or you want any changes. Here is the full list: https://prettier.io/docs/en/options.html